templates/definition/meter/volkszaehler-http.yaml

```diff
@@ -10,11 +10,14 @@ params:
   - name: host
   - name: port
     default: 80
+  - name: urlprefix
+    default: /api/
```

Comment:
bitte

    /api

Reply:
Der Parameter braucht auf jeden Fall auch eine Erläuterung.

Reply:
Wie kann man dann leer eingeben? Um `zaehler.fritz.box/data/..` zu bekommen. Wenn man im wizard Enter drückt hat man den default Wert.

Reply:
in der Art:

 ```
   description:
      de: Minimale Stromstärke in Ampere (A)
      en: Minimum amperage (A)
    help:
      de: Definiert die minimale Stromstärke pro angeschlossener Phase mit welcher das Fahrzeug geladen werden soll
      en: The minimum amperage per connected phase with which the car should be charged
 
```

Reply:
Erläuterung habe ich eingefügt. Ich finde das ist schwierig zu erklären. Mit deiner Idee einer URL als parameter, würde das meines Erachtens verständlicher (aber bleibt braeking).

Was sollen ich wegen `/api` und der Möglichkeit für dieses segment leer lassen zu machen?

Reply:
Keine Ahnung :(

Reply:
Einzig sinnvoll dürfte sein die URI einschl. `data` context einzugeben.